### ISSUE TITLE
Allow Unregistered Users to Change Theme to Dark

### ISSUE THREAD

**Issue by omarfkuri** [sublimehq/sublime_merge#1409]
### **TLDR: Dark mode should be available for unregistered users.**

I, and I assume many others, can't currently pay for the editor - or for merge. Still, Sublime Text has everything for free except for the popup, which I don't at all mind (Though I'd love to get rid of it - I'm eager to buy it when I can). Now, I see that the business model has changed. Which is great, don't get me wrong. But I do believe that some accessibility features, which are currently unavailable, are essential for all users.

**One of these features, is dark mode -  a necessary accessibility feature.**

Though I understand that some features must be exclusive for license buyers, dark mode is one that I believe is very helpful for developers' eyesight.

**Preferred solution**
Simply allow unregistered users to switch to dark, or auto mode.

**Alternatives**
Only allow dark mode. (strongly discourage this)

Thanks for reading this consideration. I love Sublime and don't see myself switching over to anything else anytime soon.

**Comment by stinos** [comment]
https://github.com/sublimehq/sublime_merge/issues/198

**Comment by aral** [comment]
As a person who has paid for Sublime Merge, I’d like to second this.

**Comment by alexchexes** [comment]
We could pay for it, but... $99 is just too high for this only feature. If it were around $20, many of us would have paid for it a few years ago. It's also worth considering that not everyone lives in a country where a typical developer's income not less than $5000/month.

**Comment by themilkman** [comment]
> We could pay for it, but... $99 is just too high for this only feature.

FWIW, just to clarify: There is a basic misunderstanding on your side I guess. While  evaluation time is not limited (which is only a kind  gesture of SHQ), usage without a license does not mean you're using a "free" version. It is what it says, an evaluation phase, but for constant usage, you'd have to get a license. And you'll get the dark mode included as a "bonus".

**Comment by alexchexes** [comment]
@themilkman 
I understand that, but I think it's worth noting that Sublime Text, and probably Merge, have operated like this for many years, leading the community to generally perceive them as free products, at least for now. Given this, it’s fair to say that many people simply want to "enable the dark theme" and $99 is too high price for that, causing many to opt out. As I mentioned, if it were $20, I know at least three other people who would have bought it years ago, but at $99, they probably never will. So instead of earning $20 x 3, SHQ decided to hold out for $99 from fewer people. That’s their choice, but I think they might earn more if they went with the first option.

**Comment by Rishi-Prasanna** [comment]
> > We could pay for it, but... $99 is just too high for this only feature.
> 
> FWIW, just to clarify: There is a basic misunderstanding on your side I guess. While evaluation time is not limited (which is only a kind gesture of SHQ), usage without a license does not mean you're using a "free" version. It is what it says, an evaluation phase, but for constant usage, you'd have to get a license. And you'll get the dark mode included as a "bonus".

No. Keeping dark mode as a "bonus" instead of a key accessibility feature is an outdated mindset.
Sublime Merge is hiding a key accessibility feature behind a paywall, which is an automatic no-thank-you from me personally. I uninstalled Sublime Merge and I'm now using another option. It really is that serious.
I'm a vision-impaired dude that relies on dark mode to even get around most interfaces. Light mode is quite literally a health risk, as my eyes are extra-sensitive to the UV rays and light.

**Comment by renegade93** [comment]
$99 is indeed a steep price. Many folks are saying they'd pay $20 but I think if it were $49 many people would've still purchased it, myself included. Please consider lowering the price of this tool.

**Comment by countzero** [comment]
+1

**Comment by Frizi** [comment]
I'm currently evaluating this software for my use. I'm happy to pay if it ends up fitting my workflow. What immediately put me off was the flashbang I received after opening it. I was thinking "Odd default choice, but to each their own. Surely it's an accessibility setting that I can adjust". Unfortunately I was wrong. I'm not able to properly evaluate it as is, since it's not usable for me in current form. It harshly clashes with my whole existing system. Color theming possibilities can be a deciding factor whether one wants to use a piece of software, so in my opinion this limitation defeats the purpose of evaluation period. I'd be fine with popups on startup or other "interruptions" when trying to use rarer features, as that doesn't prevent me from seeing the software in action. If you insist that this option should remain "exclusive", I'd consider allowing theme change **without persisting the setting**, so that the user would have to change it on every startup. That at least would let me evaluate it, even if a bit annoying.

**Comment by countzero** [comment]
Dark mode is possible for unregistered users with a little help of AI 😉 

See https://github.com/countzero/sublime_merge_dark for an installer. If something changes in the future the findings along the way are documented in https://github.com/countzero/sublime_merge_dark/blob/main/AGENTS.md.

Happy coding!

<img width="1672" height="867" alt="Image" src="https://github.com/user-attachments/assets/b018748d-0c74-4e66-8e18-33c211baa008" />
